build-android

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+#
+# build-android
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+set -e
+
+cd "$(dirname $0)/.." || exit 1
```

Comment:
Small observation; this changes the directory you're currently in. If you wanted to just find out what directory you are in, you can do it in parenthesis e.g.

https://github.com/apple/swift/blob/cb3fcd4b83410f93558bb449695e1f1c2e69cfb9/utils/swift_build_support/test_swift_build_support.sh#L5